AdyenActions/Components/3DS2/ThreeDS2Component.swift

```diff
@@ -22,7 +22,7 @@ public final class ThreeDS2Component: ActionComponent {
     /// The delegate of the component.
     public weak var delegate: ActionComponentDelegate?
 
-    /// Delegates `PresentableComponent`'s presentation.
```

Comment:
Is there a place where we could assert if the `presentationDelegate` is not set?

e.g. we do following in the `AwaitComponent`

```
let message = "PresentationDelegate is nil. Provide a presentation delegate to AwaitComponent."
AdyenAssertion.assertionFailure(message: message)
```

Reply:
We have on use - in the ThreeDS2PlusDAScreenPresenter.  Isn't that enough? 

https://github.com/Adyen/adyen-ios/blob/3d5f181ee0046da7134a64290780c30b40e38f2a/AdyenActions/Components/3DS2/Action%20handlers/3DS2%2BDelegated%20Authentication/ThreeDS2PlusDAScreenPresenter.swift#L59